README.md

```diff
@@ -1,5 +1,11 @@
-Takeaway Challenge
+Takeaway Challenge :hamburger: :pizza: :fries:
```

Comment:
I think you could add some more information in your readme. For example, you could include:

-  an explanation of the approach you took for each user story
- an example feature test in irb for each user story
- any reflections from completing the challenge e.g What was difficult? What gaps in your learning did you notice?

Reply:
Totally agree that the README.md could use more detail. I'll update shortly